ci-operator/step-registry/configure-registry-storage/nutanix/configure-registry-storage-nutanix-ref.yaml

```diff
@@ -0,0 +1,15 @@
+ref:
+  as: configure-registry-storage-nutanix
+  from_image:
+    namespace: ocp
+    name: "4.12"
+    tag: upi-installer
```

Comment:
If not necessary, suggest not use upi-installer, I guess you can use `from: cli` image.

Reply:
@jianlinliu  You mean update as below, right?
```
ref:
  as: configure-registry-storage-nutanix
  from: cli
  commands: configure-registry-storage-nutanix-commands.sh
  resources:
```

Reply:
right